.github/workflows/test-suite.yaml

```diff
@@ -133,14 +133,19 @@ jobs:
       if: |
         (steps.list-changed.outputs.changed == 'true' && !contains(github.event.pull_request.labels.*.name, 'next-release')) ||
         (contains(github.event.pull_request.labels.*.name, 'needs-testing'))
-      run: make -j8 -O fill-test-ci-values TESTS=[${{ matrix.test }}]
+      run: |
+        make install-lagoon-dependencies
+        make install-lagoon
```

Comment:
Rather than requiring two runs, could these targets be consolidated into `make install-lagoon INSTALL_LAGOON_DEPENDENCIES=true`? I thought that might make it clearer that you can toggle the behaviour rather than knowing that you need to run these commands in order.